### PR TITLE
Doc: Improve contributing documentation

### PR DESCRIPTION
**Pull request by guerinoni**
In the doc it mentions to go to subfolder of examples but it was not
there :).

Signed-off-by: Federico Guerinoni <guerinoni.federico@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ are always appreciated!
 You can use Docker and docker-compose to test shuttle locally during development. See the [Docker install](https://docs.docker.com/get-docker/)
 and [docker-compose install](https://docs.docker.com/compose/install/) instructions if you do not have them installed already.
 
+```bash
+git clone git@github.com:shuttle-hq/shuttle.git
+cd shuttle
+git submodule init
+git submodule update
```
Comment (chesedo) [inline CONTRIBUTING.md:20]: Nice! Maybe only these two lines should be in a section/heading about checking out the examples submodule?

**Review by chesedo — COMMENTED** [review]
Nice catch, thanks for the update @guerinoni

File: CONTRIBUTING.md (hunk 1 above)
Comment (guerinoni) [reply CONTRIBUTING.md]: I move this close to `examples` section :)

**Comment by chesedo** [comment]
Thanks again @guerinoni

**Pull request merged**